prow/cluster/jobs/istio/test-infra/istio.test-infra.master.gen.yaml

```diff
@@ -127,6 +127,8 @@ postsubmits:
     - ^master$
     cluster: test-infra-trusted
     decorate: true
+    labels:
+      preset-service-account: "true"
```

Comment:
I disagree with using presets, please just add the concrete fields to the pods.

Reply:
I will update the generation logic. It might be more appropriate to break this out into another PR, since the change will affect several jobs unrelated to this change..